Server/JokesOnYou.Web.Api/Services/JokesService.cs

```diff
@@ -10,10 +14,42 @@ namespace JokesOnYou.Web.Api.Services
     public class JokesService : IJokesService
     {
         private readonly IJokesRepository _jokesRepo;
+        private readonly IUserRepository _userRepository;
+        private readonly IUnitOfWork _unitOfWork;
+        private readonly IMapper _mapper;
 
-        public JokesService(IJokesRepository jokesRepo)
+        public JokesService(IJokesRepository jokesRepo, IUserRepository userRepository, IUnitOfWork unitOfWork, IMapper mapper)
         {
             _jokesRepo = jokesRepo;
+            _mapper = mapper;
+            _userRepository = userRepository;
+            _unitOfWork = unitOfWork;
+        }
+        public async Task<JokeReplyDto> CreateJokeAsync(JokeCreateDto jokeCreateDto, string userId)
+        {
+
+            var user = await _userRepository.GetUserAsync(userId);
```

Comment:
should this get the user like this or ask the UserService to get it ?
I'm guessing this is good.

Reply:
Yea its fine. Didnt see this until now github.com likes to hide things lol